### ISSUE TITLE
[1.15.2] infoSlimeChunk render not consistent

### ISSUE THREAD

**Issue by Damagedxd** [maruohon/minihud#38]
when i log out and back in and put the same exact seed the slime chunk viewer changes location This could also be because of me going back and fourth in the nether. The first picture is my first log in for the day and it don't show that the chunks in front of me that I've dug out are slime chunks, while the second does. The only factors are me logging out and back in, and going through the nether.
![1st test](https://user-images.githubusercontent.com/45430265/78236110-de07cd80-74d9-11ea-8ed6-b8aa51edfa32.png)
![2nd test](https://user-images.githubusercontent.com/45430265/78236113-dfd19100-74d9-11ea-9acb-72380843de15.png)

**Comment by maruohon** [comment]
I think the current 1.13.2+ versions have a bug where the "is the seed valid" status isn't properly cleared when logging out. I rewrote that code at one point in the main development version (1.12.2) but that code isn't yet in the other versions.

So to clarify, if you manually enter the seed after logging in, then the renderer is correct? I don't remember if you need to also toggle it off/on to refresh it after inputting the seed... Yeah basically it was buggy and it was fixed in the main development version already...

**Comment by Damagedxd** [comment]
Yes when i log in and enter seed it works. But as soon as i change dimensions it perhaps uses the wrong seed? I'm not sure, i haven't been able to fully recreate it.
